### ISSUE TITLE
Add accessibility label for precipitation strip

### ISSUE THREAD

**Issue by geosaaga** [fmidev/weather-app#1197]
Precipitation strip is missing accessibility label and AVI has noted this in their report. Define and implement accessibility label for precipitation strip that describes raining and precipitation amount during the day.
